permission/v1/contract.go

```diff
@@ -71,6 +75,10 @@ func (i *Init) GetNumberOfAccounts() (*big.Int, error) {
 	return i.permAcctSession.GetNumberOfAccounts()
 }
 
+func (i *Init) GetWhitelistedContracts() ([]common.Address, error) {
+	return []common.Address{}, nil
```

Comment:
Since this is not supported in v1, should this also return an error?
```suggestion
	return nil, fmt.Errorf("contract whitelist not supported in enhanced permissioning v1")
```